src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -347,15 +358,22 @@ private unsafe void Run()
                             _debugRecorder.Start();
                         }
                     }
+                    catch (LockAlreadyDisposedException)
```

Comment:
Better to unify all these throwing exception blocks and add a comment on why you are "handling" it in this way

Reply:
done